packages/cli/src/commands/styles/command.ts

```diff
@@ -54,32 +56,55 @@ export const styles: CommandModule<Options, Options> = {
       {
         name: 'ds-styles-dir',
```

Comment:
Can we set default for this?

Reply:
They might have more than 1 projects created, I think it is safer to ask